i18n/locales/urdu/translations.json

```
@@ -11,7 +11,11 @@
     "see-results": "See all results for {{ searchQuery }}",
     "no-tutorials": "No tutorials found"
   },
-  "banner": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
+  "banner": {
+    "default": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
```

Sorry, didn't have this one translated like the other two strings. Based on #273, this should be:

```suggestion
    "default": "{{ <0> }}کوڈ کرنا سیکھیں 3000 گھنٹوں کا مفت نصاب{{ </0> }}",
```